### PR TITLE
Create Dework.md

### PR DESCRIPTION
**Pull request by 0xPuffin**
dework is another job board for gig work at daos

### CODE REVIEW

File: data/Dework.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+```
+name: Dework
+description: Job board to do gigs for DAOs
+token_image_url:
+ticker:
+website_url: https://dework.xyz/
+webapp_url:https://app.dework.xyz/
+contract_url: https://twitter.com/_Lonis_
```
Comment (0x4C756B65) [inline data/Dework.md:8]: just need to remove this and it's good to merge!

Comment (0x4C756B65) [reply data/Dework.md:8]: L#8 `contract_url`

Comment (pentcle) [reply data/Dework.md:8]: have added it to the db for next push

Comment (0xPuffin) [reply data/Dework.md:8]: ty

**Pull request merged**